tmail-backend/apps/distributed/src/test/java/com/linagora/tmail/james/app/CombinedIdentityJamesServerTest.java

```diff
@@ -36,6 +37,7 @@ public class CombinedIdentityJamesServerTest {
         .extension(new CassandraExtension())
         .extension(new RabbitMQExtension())
         .extension(new LdapExtension())
+        .extension(new RedisExtension())
```

Comment:
Would have thought the other way around, default implem rabbitmq so those tests don't need to be changed, and would need to create some extra tests with redis pub/sub?

Reply:
That is fine to me too.

My thought when I did that: I want to make sure the Redis one functions well across our ITs. While the test with RabbitMQ one should be covered by tons of tests on James test suite.

Reply:
Not a bad thought either :)